packages/moonstone/BodyText/BodyText.js

```diff
@@ -11,16 +11,20 @@
  */
 
 import kind from '@enact/core/kind';
-import UiBodyText from '@enact/ui/BodyText';
 import React from 'react';
 import PropTypes from 'prop-types';
 import compose from 'ramda/src/compose';
+import UiBodyText from '@enact/ui/BodyText';
 import Pure from '@enact/ui/internal/Pure';
 
+import {MarqueeDecorator} from '../Marquee';
```

Comment:
Wanted to note here that `ui/Marquee` doesn't have any i18n support which is added by the `moonstone` version. We should consider how that support will be added to `moonstone/BodyText`.

Reply:
nvm ... this is the moonstone version ... :nothingtodohere: ...